day-to-day.md

```diff
@@ -177,3 +169,25 @@ Our goal is to be prepared for client meetings 15 minutes before they begin, reg
 ### Remote Meetings
 
 We use [Zoom](https://zoom.us/) for remote meetings. You can sign-up and use Zoom for free with your Base Two email address. The free version works great for 1-1 video calls, but is limited to 40 minutes when there are multiple attendees. We have a paid account through info@base2.io, which is useful for client meetings. The password for the paid account can be found in the [credentials document](https://b2io.slack.com/archives/C904HLD5L/p1516982426000280).
+
+### Base Two Day
+
+Base Two Day is celebrated annually on August 26. It is the anniversary of the founding of Base Two. In the past, this day has been celebrated by hanging out, or participating in Hackathons. Tracking of what has been done on this day is now kept on the team's Google Drive. Discussions around this topic can be found on the [#side-quests](https://join.slack.com/share/zt-h7xz6k8g-FuetGuYYeuKYj34ukKptCg) Slack channel. Base Two Day is also a day when we try to remember to review the handbook!
+
+## We Use
```

Comment:
Flagging so that everyone has a chance to add any software / tool recommendations.